pkg/deploy/deploy.go

```diff
@@ -41,42 +41,17 @@ func DeployConfigs(client dtclient.Client, apis api.APIs, sortedConfigs []config
 	entityMap := newEntityMap(apis)
 	var errors []error
 
-	for _, c := range sortedConfigs {
-		c := c // to avoid implicit memory aliasing (gosec G601)
+	for i := range sortedConfigs {
+		c := &sortedConfigs[i] // avoid implicit memory aliasing (gosec G601)
 
 		if c.Skip {
```

Comment:
This is now duplicated - the depoy method as well as the DeployConfigs logic now do check if Skip is set. Imo, the code, as it was, was fine. 
Now we have two places to worry. 

With this change, we no do
1) Info log when skipping
2) Printing we do validation or deployment (we aren't, we skip the config!)
3) Return an empty object if we skip

The code is not optimal, but I suggest moving the continue back in this method. Skipping really means no deployment. I like moving the deploy functionality in its own method, but skipped configs should never reach it.

Reply:
The code is far from optimal, on that we agree :))

I'll move the logging info in `deploy` function, as well as logging for "real" deployment..

Bullet 3) I do not understand.
Functionality with the refactoring stays the same as before. Only problem is with logging (on info level). But that was also before, e.g. for `case config.EntityType` first prints on the info level that something is deployed, and after that on debug level that it is not.